### PR TITLE
Feature/reset form

### PR DESCRIPTION
**Pull request by davekwon555**
## Add "Reset" button to each form ##
* If a "New" form, "Reset" will clear out all text fields and set select boxes(select, radio button, checkbox, etc.) to default value
* If an "Edit" form, "Reset" will reset each field to pre-populated values

**Note: Currently the forms do not have correct connections between label and input field. This prevented Capybara from finding a correct input field to fill in. I have left @wip tag in the feature so that Travis CI does not fail. @csungwon will fix the form selectors very soon**

### CODE REVIEW

**Review by kyriakipapadimitriou — APPROVED** [review]
Everything looks good!

File: features/clear_form.feature [hunk 1]
```diff
@@ -163,95 +162,91 @@ Scenario: user can reset a form for Edit App
...
+	And I fill in "some comment" for "Satisfaction Comments"
+	And I press "Reset"
+	Then I should have filled in "2017-11-27" for "End Date"
+	And the "Duration" field should contain "15 min"
+	And the "Demeanor" field should contain "Strongly agree"
+	And the "Engagement" field should contain "Strongly agree"
+	And the "Communication" field should contain "Strongly agree"
+	And the "Understanding" field should contain "Strongly agree"
+	And the "Effectiveness" field should contain "Strongly agree"
+	And the "Satisfaction" field should contain "Strongly agree"
```
Comment (Jldeatrick) [inline features/clear_form.feature:243]: These could probably be refactored into the standard hash table format from other feature files, but they are very extensive tests! Good job.

**Review by Jldeatrick — APPROVED** [review]
Looks great! Good work taking care of the various bootstrap changes. Overall great UX improvement

File: features/clear_form.feature (hunk 1 above)
Comment (kimhoang73) [reply features/clear_form.feature:243]: agree with Joe here!

**Review by kimhoang73 — APPROVED** [review]
looks good, these are nice changes to make the app more understandable later

**Review by julies642 — APPROVED** [review]
Looks great! Thanks for the fix ups and extra tests.

**Pull request merged**